packages/fluentui/react-bindings/src/index.ts

```diff
@@ -25,6 +25,9 @@ export * from './hooks/useStyles';
 export { useTriggerElement } from './hooks/useTriggerElement';
 export { useUnhandledProps } from './hooks/useUnhandledProps';
 export { usePrevious } from './hooks/usePrevious';
+export { useFrameFocusDispatch } from './hooks/useFrameFocusDispatch';
+export { useFrameListener } from './hooks/useFrameListener';
```

Comment:
I don't think you need to export these unless there is a scenario where they explicitly should be used separately